omegaconf/_utils.py

```diff
@@ -593,6 +598,20 @@ def is_tuple_annotation(type_: Any) -> bool:
         return origin is tuple  # pragma: no cover
 
 
+def is_literal_annotation(type_: Any) -> bool:
+    origin = getattr(type_, "__origin__", None)
+    if sys.version_info >= (3, 8, 0):
```

Comment:
to be consistent with the others
```suggestion
    if sys.version_info >= (3, 8):
```